packages/protocol/contracts/L1/libs/LibProposing.sol

```diff
@@ -138,7 +137,8 @@ library LibProposing {
                         config.proposerRewardMax
                     );
 
-                    state.taikoTokenBalances[input.beneficiary] += reward;
+                    // Reward must be minted
+                    tt.mint(input.beneficiary, reward);
```

Comment:
Just wondering should we use `msg.sender` (a L1 address) instead of `input.beneficiary` (a L2 address) here? since `msg.sender` is the account who paid the ether fees here.

Reply:
If the fee is smaller than the reward, then it will be a problem. So maybe it is safer to use input.beneficiary instead of msg.sender.